{{cookiecutter.project_slug}}/clients/mobile/react-native/src/screens/routes.ts

```diff
@@ -28,10 +29,15 @@ export const tabDefaultOptions = (): BottomTabNavigationOptions => ({
 })
 // NAVIO
 export const navio = Navio.build({
-  screens: { Auth, Login, SignUp, Main, DashboardScreen },
+  screens: { Auth, Login, SignUp, Main, DashboardScreen, ComponentsPreview },
   stacks: {
     AuthStack: ['Auth'],
     MainStack: ['DashboardScreen'],
+    /**
+     * Set me as the root to see the components preview
+     */
+    PreviewStack: ['ComponentsPreview'],
```

Comment:
I think for new projects we should just link to this view from the main screen. It'll then be up to the devs to remove it, hide it, etc.
As it is, I'm concerned it'll be lost/forgotten

Reply:
I was thinking we would just add it to the Main landing page with a new auth screen that we used in other projects, that way its a big clear button that says To the Components Preview!